process.sh

```diff
@@ -5,58 +5,58 @@ die() { set +v; echo "$*" 1>&2 ; exit 1; }
 
 if [ "$#" -ne 0 ]; then
     die 'Collects source data, processes it, and pushes to S3.
-No commandline arguments, but looks for two environment variables:
-  "NO_PUSH=true" will fetch and process data, but not push to S3.
-  "CI=true" will use fixtures, rather than fetching, and also will not push to S3.
-'
+    No commandline arguments, but looks for two environment variables:
+    "NO_PUSH=true" will fetch and process data, but not push to S3.
+    "CI=true" will use fixtures, rather than fetching, and also will not push to S3.
+    '
 fi
 
 main() {
-  process_linnarson_cells
-  process_linnarson_molecules
-  process_linnarson_images
-  process_giotto
-  process_mermaid
-
-  echo
-  echo 'input:'
-  ls -lh "$INPUT"/*
-
-  echo
-  echo 'output:'
-  ls -lh "$OUTPUT"/*
-
-  echo
-  echo 'AWS:'
-  if [[ "$CI" = 'true' ]] || [[ "$NO_PUSH" = 'true' ]]
-  then
-    echo 'CI: Skip push to AWS'
-  else
-    # Exclude the *HUGE* PNGs in the base directory:
-    # The tiles for S3 are in subdirectories;
-    # We keep the PNGs around because it takes a long time to generate them.
-    aws s3 cp --exclude "$OUTPUT/*.png" --recursive "$OUTPUT" s3://"$S3_TARGET"
-  fi
+    echo
+    echo 'Input:'
+    ls -lh "$INPUT"/
+
+    echo
+    echo 'Output:'
+    ls -lh "$OUTPUT"/
+
+    echo
+    echo "Processing Linnarsson data"
+    ./scripts/process_linnarsson.sh -b "$BASE" -i "$INPUT" -o "$OUTPUT" -t "$S3_TARGET"
+
+    echo
+    echo "Processing Dries data"
+    ./scripts/process_dries.sh -b "$BASE" -i "$INPUT" -o "$OUTPUT" -t "$S3_TARGET"
+
+    echo
+    echo "Processing Wang data"
+    ./scripts/process_wang.sh -b "$BASE" -i "$INPUT" -o "$OUTPUT" -t "$S3_TARGET"
+
+    echo 'AWS:'
+    if [[ "$CI" = 'true' ]] || [[ "$NO_PUSH" = 'true' ]]
+    then
+        echo 'CI: Skip push to AWS'
+    else
+        # Exclude the *HUGE* PNGs in the base directory:
+        # The tiles for S3 are in subdirectories;
+        # We keep the PNGs around because it takes a long time to generate them.
+        aws s3 cp --exclude "$OUTPUT/*.png" --recursive "$OUTPUT" s3://"$S3_TARGET"
```

Comment:
You had asked how we should handle the really-big-ometiff-that-nothing-actually-uses: I had forgotten this line, which excludes some files from the push to s3.